docs/introduction.md

```diff
@@ -41,7 +41,9 @@ npm install --save amphora
 
 ## Usage
 
-Clay separates concerns into two main areas: components and sites. Create two new directories in your project:
+Clay separates concerns into two main areas: components and sites. 
```

Comment:
```suggestion
Clay separate concerns into two main areas: components and sites. 
```

Reply:
Nice suggestion Steph, but the correct way for singular would be as it was before. So, it is good as it is right now